src/gluonts/mx/__init__.py

```diff
@@ -24,3 +24,11 @@
 Tensor = typing.Union[mx.nd.NDArray, mx.sym.Symbol]
 
 from . import prelude as _  # noqa
+from .batchify import as_in_context, batchify  # noqa
+from .block.scaler import MeanScaler, NOPScaler  # noqa
+from .distribution import DistributionOutput, GaussianOutput  # noqa
+from .kernels import RBFKernel  # noqa
+from .model.estimator import GluonEstimator  # noqa
+from .model.predictor import RepresentableBlockPredictor  # noqa
+from .trainer import Trainer  # noqa
+from .util import copy_parameters, get_hybrid_forward_input_names  # noqa
```

Comment:
Something we could do is to define an `__all__`, which would allow us to remove the noqa-comments.